requirements/edx/custom.txt

```diff
@@ -21,7 +21,7 @@
 -e git+https://github.com/mckinseyacademy/xblock-diagnosticfeedback.git@v0.2.4#egg=xblock-diagnostic-feedback==0.2.4
 -e git+https://github.com/open-craft/xblock-group-project-v2.git@0.4.9#egg=xblock-group-project-v2==0.4.9
 -e git+https://github.com/open-craft/xblock-virtualreality.git@v0.1.1#egg=xblock-virtualreality==0.1.1
-git+https://github.com/edx-solutions/api-integration.git@v1.8.5#egg=api-integration==1.8.5
+git+https://github.com/edx-solutions/api-integration.git@kshitij/port-completion-v1#egg=api-integration==2.0.0
```

Comment:
:+1: pending the version bump here when that lands on api-integration.

Reply:
Also pending resolution of the failing tests, of course.

Reply:
I got all the tests to pass, these seem to have popped up post rebase, so not sure what's gone wrong. Will look into it.

Reply:
Thanks.  Not sure I understand either.  It looks clean.

Reply:
I think I messed up somewhere and a change I had reverted got reintroduced here. I've fixed it and now the tests are passing with the [latest commit](https://github.com/edx-solutions/api-integration/pull/138/commits/3e65826ac18609fa1edb5f51aac3320e0a1b3860)